docs/GetStarted/cmake_options_and_variables.md

```diff
@@ -1,74 +1,96 @@
-# IREE-specific CMake Options and Variables
+# CMake Options and Variables
+
+## Frequently-used CMake Variables
+
+#### `CMAKE_BUILD_TYPE`:STRING
+
+Sets the build type. Possible values are `Release`, `Debug`,
+`RelWithDebInfo`/`FastBuild` and `MinSizeRel`. If unset, build type is set to
+`Release`.
+
+#### `CMAKE_<LANG>_COMPILER`:STRING
+
+This is the command that will be used as the `<LANG>` compiler, which are `C`
+and `CXX` in IREE. These variables are set to compile IREE with `clang` or
+rather `clang++`. Once set, these variables can not be changed.
+
+## IREE-specific CMake Options and Variables
 
 This gives a brief explanation of IREE specific CMake options and variables.
 
-*   `IREE_ENABLE_RUNTIME_TRACING`:BOOL
+#### `IREE_ENABLE_RUNTIME_TRACING`:BOOL
+
+Enables instrumented runtime tracing. Defaults to `OFF`.
 
-    Enables instrumented runtime tracing. Defaults to `OFF`.
+#### `IREE_ENABLE_MLIR`:BOOL
 
-*   `IREE_ENABLE_MLIR`:BOOL
+Enables MLIR/LLVM dependencies. Defaults to `ON`. MLIR/LLVM dependencies are
+required if the IREE compiler should be build. Therefore, the option is
```

Comment:
```suggestion
required when building the IREE compiler components. Therefore, the option is
```

nit: "build" -> "built", or reword (suggested one rephrasing)

Reply:
Thanks for the review and the suggestions.